skyportal/tests/api/test_annotations_on_spectrum.py

```diff
@@ -79,7 +79,7 @@ def test_add_and_retrieve_annotation_group_id(
         token=annotation_token,
     )
 
-    assert status == 400
+    assert status in [500, 401]
     assert 'duplicate key value violates unique constraint' in data["message"]
```

Comment:
500 is when system is revisioning? Would the next assert fail with a different `data['message']` in that case?

Reply:
We should probably get rid of the 500.